<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PinFunctionsDialog</class>
 <widget class="QDialog" name="PinFunctionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>247</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>301</width>
    <height>247</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>301</width>
    <height>247</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GPIO Pin Functions</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icons/active64.png</normaloff>
    <normalon>:/icons/icons/active64.png</normalon>
    <disabledoff>:/icons/icons/greyed64.png</disabledoff>
    <disabledon>:/icons/icons/greyed64.png</disabledon>
    <activeoff>:/icons/icons/active64.png</activeoff>
    <activeon>:/icons/icons/active64.png</activeon>
    <selectedoff>:/icons/icons/selected64.png</selectedoff>
    <selectedon>:/icons/icons/selected64.png</selectedon>:/icons/icons/active64.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>206</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelGPIO0Value">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelGPIO0">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>65</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>GPIO.0:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelGPIO1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>27</y>
     <width>65</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>GPIO.1:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelGPIO1Value">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>27</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelGPIO2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>44</y>
     <width>65</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>GPIO.2:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelGPIO2Value">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>44</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelGPIO3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>61</y>
     <width>65</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>GPIO.3:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelGPIO3Value">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>61</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelGPIO4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>78</y>
     <width>65</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>GPIO.4:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelGPIO4Value">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>78</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelGPIO5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>95</y>
     <width>65</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>GPIO.5:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelGPIO5Value">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>95</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelGPIO6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>112</y>
     <width>65</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>GPIO.6:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelGPIO6Value">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>112</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelGPIO7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>129</y>
     <width>65</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>GPIO.7:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelGPIO7Value">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>129</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelGPIO8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>146</y>
     <width>65</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>GPIO.8:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelGPIO8Value">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>146</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelGPIO9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>163</y>
     <width>65</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>GPIO.9:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelGPIO9Value">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>163</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelGPIO10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>65</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>GPIO.10:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelGPIO10Value">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PinFunctionsDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
